api/controllers/TextController.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+
+const _ = require('lodash')
+const Controller = require('trails-controller')
+
+module.exports = class TextController extends Controller {
+
+  commandLine (request, reply) {
```

Comment:
Why the command line is build on back side ? It can easily build in the browser right ? No need to waste server resources for this